manimlib/mobject/changing.py

```diff
@@ -3,7 +3,7 @@
 from manimlib.mobject.types.vectorized_mobject import VGroup
 from manimlib.utils.rate_functions import smooth
 from manimlib.utils.space_ops import get_norm
-
+#random test
 
 class AnimatedBoundary(VGroup):
```

Comment:
Note that both let and const are block-scoped, whereas var is function-scoped. test

---
**Sema Reaction:** :trophy: This code is awesome | **Sema Tags:** Unreadable, Secure, Efficient

Reply:
this code is unreadable, not maintainable, and not secure -- fix it!

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Unreadable

Reply:
this code is unreadable, not maintainable, and not secure -- please fix it!

---
**Sema Reaction:** :hammer_and_wrench: This code needs a fix | **Sema Tags:** Unreadable

Reply:
When adding a new package, include examples of intended usage: a runnable Example, or a simple test demonstrating a complete call sequence.

Read more about testable Example() functions.

📄 [Golang.org](https://github.com/golang/go/wiki/CodeReviewComments#examples)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](http://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
manim

Reply:
test 1/18/2022


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Reusable

Reply:
test 1/19/2022

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Secure, Inefficient, Reusable

Reply:
test 1/25/2012

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Secure, Inefficient, Fault-tolerant

Reply:
test snippet 3/29/2022

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](http://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not reusable, Fault-tolerant

Reply:
test 4/7/2022

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Not reusable

Reply:
test 4/41/022

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

Reply:
test 4/15/2022

Reply:
test 2022

Reply:
test 4/15/2022 1

Reply:
test 4/15/2022 2

Reply:
test 4/15/2022 3

Reply:
ddd

Reply:
yellow panda is cool

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

Reply:
add repo to QA env

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient, Reusable

Reply:
URLs can contain dynamic script content via javascript: protocol URLs. Use validation to assure your links are http: or https: to avoid javascript: URL based script injection. Achieve URL validation using a native URL parsing function then match the parsed protocol property to an allow list.

📄 [snyk](https://snyk.io/blog/10-react-security-best-practices/)URLs can contain dynamic script content via javascript: protocol URLs. Use validation to assure your links are http: or https: to avoid javascript: URL based script injection. Achieve URL validation using a native URL parsing function then match the parsed protocol property to an allow list.

📄 [snyk](https://snyk.io/blog/10-react-security-best-practices/)

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question

Reply:
test 8/4/2022

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Readable, Not secure, Reusable

Reply:
test 8/14/2022 personal__

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome

Reply:
test 8/14/2022 org__

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Efficient

Reply:
test fix

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good&nbsp; | &nbsp;**Tags:** Readable, Not secure, Elegant, Brittle